repour/server/endpoint/endpoint.py

```diff
@@ -148,11 +162,13 @@ def do_call():
             except exception.DescribedError as e:
                 status = 400
                 traceback_id, obj = described_error_to_obj(e)
+                ERROR_VALIDATION_JSON_COUNTER.inc()
```

Comment:
@thescouser89 sorry for late notice, shouldn't it be ERROR_RESPONSE_400_COUNTER?